roles/jmeter/tasks/main.yml

```diff
@@ -28,7 +28,7 @@
 
 - name: "move the tar file's base directory somewhere maintainable"
   command:
-    mv: "{{ jmeter_tar_directory }} {{ jmeter_home_directory }}"
+    mv "{{ jmeter_tar_directory }}" "{{ jmeter_home_directory }}"
```

Comment:
Changing to use argv: will require a minimum Ansible version of V2.6. The most recent version available in the ubuntu repo is 2.5.1 so bootstrapping the whole thing on a new box will fail without the developer knowing to add the appropriate ppa before installing. Maybe we should check the ansible version at the top of the workstation.sh script (or maybe workstation.yml) and either halt or upgrade Ansible if the Ansible version is below a minimum.

Reply:
I also tested using only a single set of quotes for the parameters and it failed. The command's arguments need to be space delimited so putting both source and destination directories in a single set of quotes passes them as a single argument to mv which is looking for 2 args.

From the command-module documentation:

> The command module takes the command name followed by a list of space-delimited arguments.

Reply:
When I said a single set of quotes I meant:

```json
- name: "move the tar file's base directory somewhere maintainable"
  command: "mv {{ jmeter_tar_directory }} {{ jmeter_home_directory }}"
```

But per that paragraph of the documentation, I can see that perhaps the answer is that they shouldn't be quoted at all.  Quotes are always required if a variable expansion immediately follows the colon but otherwise, strings are assumed.

I'm getting Ansible 2.7.5 from the Ubuntu repositories - this laptop is still on Xubuntu 16.04.  Which OS are you currently using?

Reply:
Mint.
```
sgs116@sgs116-XPS ~ $ apt-cache madison ansible
   ansible | 2.7.5-1ppa~bionic | http://ppa.launchpad.net/ansible/ansible/ubuntu bionic/main amd64 Packages
   ansible | 2.7.5-1ppa~bionic | http://ppa.launchpad.net/ansible/ansible/ubuntu bionic/main i386 Packages
   ansible | 2.5.1+dfsg-1 | http://mirror.steadfast.net/ubuntu bionic/universe amd64 Packages
   ansible | 2.5.1+dfsg-1 | http://mirror.steadfast.net/ubuntu bionic/universe i386 Packages
   ansible | 2.7.5-1ppa~bionic | http://ppa.launchpad.net/ansible/ansible/ubuntu bionic/main Sources
```